### PR TITLE
[finished #94045056] Implementation autentication with facebook and t…

### PR DESCRIPTION
**Pull request by edivandecastro**
Activity finishing

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,46 +1,21 @@
 source 'https://rubygems.org'
 
-
```
Comment (edivandecastro) [inline Gemfile]: Remove line

File: Gemfile [hunk 2]
```diff
@@ -1,46 +1,21 @@
...
-  gem 'byebug'
-
-  # Access an IRB console on exception pages or by using <%= console %> in views
+	gem 'pry-byebug', '3.1.0'
   gem 'web-console', '~> 2.0'
-
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
 end
-
```
Comment (edivandecastro) [inline Gemfile]: Remove the comments

File: Gemfile [hunk 3]
```diff
@@ -1,46 +1,21 @@
...
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.0'
-# Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
-# Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
 gem 'coffee-rails', '~> 4.1.0'
```
Comment (edivandecastro) [inline Gemfile]: Remove the gem coffe-rails

File: app/assets/javascripts/welcome.coffee [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (edivandecastro) [inline app/assets/javascripts/welcome.coffee]: Remove this file

File: app/controllers/callbacks_controller.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+		@player = Player.find_for_oauth(env["omniauth.auth"], current_player)
+		if @player.persisted?
+			sign_in_and_redirect @player, event: :authentication
+			set_flash_message(:notice, :success, kind: "Facebook") if is_navigational_format?
+		else
+			session["devise.#{provider}_data"] = env["omniauth.auth"]
+			redirect_to new_player_registration_url
+		end
+	end
+end
```
Comment (edivandecastro) [inline app/controllers/callbacks_controller.rb]: Bad indentation

File: app/models/player.rb [hunk 6]
```diff
@@ -1,6 +1,28 @@
...
+      )
+      player.save!
+  	end
+
+  	if identity.player != player
+  		identity.player = player
+  		identity.save!
+  	end
+  	player
+  end
```
Comment (edivandecastro) [inline app/models/player.rb]: Bad indentetion

File: test/models/identity_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class IdentityTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (Homologator) [inline test/models/identity_test.rb]: Remove the tests

**Comment by Homologator** [comment]
Add the folder test in .gitignore

File: app/models/player.rb (hunk 6 above)
Comment (Homologator) [inline app/models/player.rb]: bad indentation

File: .gitignore [hunk 8]
```diff
@@ -14,7 +14,6 @@ pickle-email-*.html
 
 # TODO Comment out these rules if you are OK with secrets being uploaded to the repo
 config/initializers/secret_token.rb
```
Comment (Homologator) [inline .gitignore]: remove the .gitignore

File: app/controllers/welcome_controller.rb [hunk 9]
```diff
@@ -0,0 +1,5 @@
+class WelcomeController < ApplicationController
+	skip_before_filter :authenticate_player!
```
Comment (edivandecastro) [inline app/controllers/welcome_controller.rb]: bad indentation

**Comment by Homologator** [comment]
:+1:

**Pull request merged**